expression/aggregation/descriptor.go

```diff
@@ -79,6 +79,27 @@ func (a *AggFuncDesc) Clone() *AggFuncDesc {
 	return &clone
 }
 
+// Split splits `a` into two aggregate descriptors for partial phase and
+// final phase individually.
+// This function is only used when executing aggregate function parallelly.
```

Comment:
This can also be used when we split the the aggregate operator into final(executed in tidb) + partial(executed in tikv) aggregate operators.

Reply:
It's the duty of `plan/ task.go: newPartialAggregate ` now.
I think we'd better maintain the comment, it will confuse others
since we do not use this function to split root tasks and cop tasks for agg.

Reply:
OK